aequilibrae/paths/linear_approximation.py

```diff
@@ -339,7 +341,14 @@ def doWork(self):
 
     def execute(self):
         # We build the fixed cost field
+
         for c in self.traffic_classes:
+            # Sizes the temporary objects used for the results
+            c.results.prepare(c.graph, c.matrix)
+            c._aon_results.prepare(c.graph, c.matrix)
+            c.results.reset()
+
+            # Prepares the fixed cost to be used
             if c.fixed_cost_field:
                 # divide fixed cost by volume-dependent prefactor (vot) such that we don't have to do it for
                 # each occurence in the objective funtion. TODO: Need to think about cost skims here, we do
```

Comment:
@pedrocamargo this is a comment I left when we added generalised cost, how does this interact with skimming? When producing travel time skims this is not a problem but can you remind me what options we expose to generate skims?

Reply:
Skims are different arrays in the graph object, where the graph.cost/compact_cost is what is actually used in path computation.

Reply:
If we do not expose generalised cost for skimming then I don't even need to think about this. Looks good otherwise.